tools/topology/sof/tokens.m4

```diff
@@ -90,3 +90,5 @@ SectionVendorTokens."sof_tone_tokens" {
 SectionVendorTokens."sof_process_tokens" {
 	SOF_TKN_PROCESS_TYPE			"900"
 }
+
+ABI_INFO(sof_manifest, 0x03, 0x02, 0x00)
```

Comment:
@bardliao maybe it makes sense for all of this to go in a separate abi.m4 file?

Reply:
@ranj063 new version is updated.  Separated abi.m4 is created and debug message is added.